### PR TITLE
Sve4a contacts map

### PR DESCRIPTION
**Pull request by sve4a**

### CODE REVIEW

File: client/src/components/Categories/Categories.js [hunk 1]
```diff
@@ -167,5 +162,3 @@ export const Categories = () => {
     </div>
   );
 };
-
-export default Categories;
```
Comment (OlgaVakulenko) [inline client/src/components/Categories/Categories.js]: It should be here "export default Categories@

File: client/src/components/Categories/Categories.js [hunk 2]
```diff
@@ -154,7 +149,7 @@ export const Categories = () => {
   const classes = useStyles();
```
Comment (OlgaVakulenko) [inline client/src/components/Categories/Categories.js]: It should be a logic. It disappeared

File: client/src/components/Mainpage/MainPage.js [hunk 3]
```diff
@@ -2,20 +2,17 @@ import React from 'react';
 import '../../index.css';
 
 import { Header } from '../../commons';
-import Categories from '../Categories/Categories';
+import { Categories } from '../Categories/Categories';
 import BrandsBlock from './BrandsBlock';
-import StayInTouch from '../../commons/Footer/StayInTouch';
```
Comment (tfetisova) [inline client/src/components/Mainpage/MainPage.js]: We need to put StayInTouch import here

File: client/src/components/Mainpage/MainPage.js [hunk 4]
```diff
@@ -2,20 +2,17 @@ import React from 'react';
...
 
-export const MainPage = props => {
-  return (
-      <React.Fragment>
-
-          <Header callCenter={'1-855-324-5387'} />
-          <BrandsBlock />
-          <Favorites />
-          <Categories />
-          <StayInTouch />
```
Comment (tfetisova) [inline client/src/components/Mainpage/MainPage.js]: StayInTouch component deleted. It should be placed here

File: package.json [hunk 5]
```diff
@@ -31,6 +31,7 @@
     "nodemailer": "^6.3.0",
     "passport": "^0.4.0",
     "passport-jwt": "^4.0.0",
+    "react-google-maps": "^9.4.5",
```
Comment (antilik) [inline package.json]: is it necessary to install it on the back end?

Comment (antilik) [reply package.json]: finally it was deleted by sve4a.

**Pull request merged**